DrrImageComputation/Widgets/qSlicerDrrImageComputationPlastimatchParametersWidget.cxx

```diff
@@ -317,8 +315,9 @@ void qSlicerDrrImageComputationPlastimatchParametersWidget::onHUThresholdChanged
     qCritical() << Q_FUNC_INFO << ": Invalid parameter node";
     return;
   }
-
+  d->ParameterNode->DisableModifiedEventOn();
```

Comment:
Is this one necessary?

Reply:
It's recommended. Every time when the HU threshold value is changed, the vtkCommand::ModifiedEvent is invoked and it updates markups geometry, which it quite slow. In that case GUI update becomes sluggish.